src/Language/PureScript/Sugar/Operators/Common.hs

```diff
@@ -47,7 +47,6 @@ opTable
   -> [[P.Operator (Chain a) () Identity a]]
 opTable ops fromOp reapply =
   map (map (\(name, a) -> P.Infix (P.try (matchOp fromOp name) >> return (reapply name)) (toAssoc a))) ops
-  ++ [[ P.Infix (P.try (parseOp fromOp >>= \ident -> return (reapply ident))) P.AssocLeft ]]
```

Comment:
Did the change from #3212 accidentally sneak in here? ;) https://github.com/purescript/purescript/pull/3212/files

Reply:
It didn't sneak in no, it's kinda required since `reapply` needs a source position now, and this case can't provide one (unless / until the entire AST has source positions).

I've been considering how to test it since seeing Phil's reply to #3212, as it didn't break any of the existing tests.

Reply:
Oh, er, maybe it snuck into this specific PR after all, but one of the others I opened about the same time makes the above comment true.